recipes-tag/12-id-smi-collection/meta.yaml

```diff
@@ -0,0 +1,14 @@
+{% set year = "2018" %}
+{% set cycle = "2" %}
+{% set version ="0" %}
+
+package:
+  name: 12-id-smi-collection
+  version: {{ year }}-{{ cycle }}.{{ version }}
```

Comment:
same here.

Reply:
Thanks @licode for the hint! Do we want another PR taking care of it? Or reverting and fixing may be better?

Reply:
I will revert it.